### PR TITLE
Patch image digest of ingress-nginx

### PR DESCRIPTION
**Pull request by supersven**
Image digests are not supported by our `wire-server-deploy` machinery. They cannot be correctly dumped to tar files and then be loaded by `containerd`.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by supersven** [comment]
Likely overtaken by https://github.com/wireapp/wire-server-deploy/pull/602/commits/8b9e1cae427c91b285142da3d18449d95b407d35

File: charts/sftd/Chart.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ version: 0.0.42
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 3.1.3
+appVersion: 3.1.13
```
Comment (jschaul) [inline charts/sftd/Chart.yaml:14]: perhaps this change should be in its separate PR?

**Review by jschaul — APPROVED** [review]
Looks fine.

**Comment by supersven** [comment]
Overtaken by events: This happens now in `wire-server-deploy`.
